src/lib.rs

```diff
@@ -180,23 +176,23 @@ pub struct ServerLoginStartParams {
 
 #[derive(Debug, Serialize, Deserialize, Tsify)]
 #[tsify(into_wasm_abi, from_wasm_abi)]
-pub struct ServerLoginStartResult {
-    #[serde(rename = "serverLogin")]
-    server_login: String,
-    #[serde(rename = "credentialResponse")]
-    credential_response: String,
+pub struct StartServerLoginResult {
+    #[serde(rename = "serverLoginState")]
+    server_login_state: String,
+    #[serde(rename = "loginResponse")]
+    login_response: String,
 }
 
-#[wasm_bindgen(js_name = serverLoginStart)]
-pub fn server_login_start(
-    params: ServerLoginStartParams,
-) -> Result<ServerLoginStartResult, JsError> {
+#[wasm_bindgen(js_name = startServerLogin)]
+pub fn start_server_login(
+    params: StartServerLoginParams,
+) -> Result<StartServerLoginResult, JsError> {
     let server_setup = decode_server_setup(params.server_setup)?;
-    let password_file_bytes = match params.password_file {
+    let password_file_bytes = match params.registration_record {
         Some(pw) => base64_decode("passwordFile", pw).map(Some),
         None => Ok(None),
     }?;
-    let credential_request_bytes = base64_decode("credentialRequest", params.credential_request)?;
+    let credential_request_bytes = base64_decode("credentialRequest", params.start_login_request)?;
```

Comment:
instead of `credentialRequest` we probably should have `startLoginRequest`?

Reply:
yeah right, I was just going through the react-native implementation and I realized I forgot actually to change these names for the error messages.

Reply:
Should be fixed now with latest commit.